### ISSUE TITLE
feat: empty state component

### ISSUE THREAD

**Issue by jotatoledo** [Teradata/covalent#991]
An empty state component according to the material design guidelines

##### Screenshots or link to CodePen/Plunker/JSfiddle

An example implemented for vue can be found in the following link:
https://vuematerial.io/components/empty-state

#### What is the motivation / use case for changing the behavior?

Handling empty states after filtering, search or similar operations is a really standard use case, so having something like this could easily be reused by many devs.

**Comment by aciccarello** [comment]
Related to #968

**Comment by emoralesb05** [comment]
Yeah, we are working on lots of design patterns going forward that can potentially be converted into components 😄

**Comment by kyleledbetter** [comment]
Yup we actually already started on that pattern https://stackblitz.com/edit/empty-states

Not sure it's worth the overhead of a component since it's lightweight markup

**Comment by jotatoledo** [comment]
@kyleledbetter I agree, its indeed a really simple element, But Im my opinion, adding an standard element to be reused in covalent is a gain and enforces a standard, as any developer could create their own version of it, which could not stick to the material specs.

**Comment by adamnel** [comment]
Does it have to be a full component? How about we provide the markup to copy/paste and some common variations, with a little explanation about when and how to use them?
